website/docs/r/spring_cloud.html.markdown

```diff
@@ -0,0 +1,69 @@
+---
+subcategory: "App Platform"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_spring_cloud"
+description: |-
+  Manages an Azure Spring Cloud Service.
+---
+
+# azurerm_spring_cloud
+
+Manages an Azure Spring Cloud Service.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "Southeast Asia"
+}
+
+resource "azurerm_spring_cloud" "example" {
+  name                = "sc-example"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+
+  tags = {
+    Env = "staging"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the Service resource. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal. Changing this forces a new resource to be created.
+
+* `location` - (Required) The GEO location of the resource. Changing this forces a new resource to be created. Until November 20, 2019 4:20 PM, only `East US`, `Southeast Asia`, `West Europe`, `West US 2` following locations are supported.

```

Comment:
can we update this to match the terminology used by other resources here?

the "only supported in X regions" can be achieved using an info box:

```
-> **Note:** At this time Azure Spring Cloud is only supported in a subset of regions (including `East US`, `South East Asia`, `West Europe` and `West US 2`.
```